pkg/consensus/recovery_message.go

```diff
@@ -187,16 +191,14 @@ func (m *recoveryMessage) AddPayload(p payload.ConsensusPayload) {
 		m.changeViewPayloads = append(m.changeViewPayloads, &changeViewCompact{
 			ValidatorIndex:     p.ValidatorIndex(),
 			OriginalViewNumber: p.ViewNumber(),
-			Timestamp:          p.GetChangeView().Timestamp(),
+			Timestamp:          p.GetChangeView().(*changeView).timestamp,
```

Comment:
And `Timestamp` returns the same thing now.

Reply:
It returns 64-bit value, but we need 32-bit for a `Timestamp` in CVCompact payload. The other way to solve it here is by `/ nanoInSec`, but that would mean multiplying (in `Timestamp()`) and then dividing for no good reason.